.codeclimate.yml

```diff
@@ -10,7 +10,7 @@ checks:
     enabled: false
   method-complexity:
     config:
-      threshold: 6
+      threshold: 15
```

Comment:
To match our Rubocop setting. Code Climate was complaining that `send_sms` was too complex. I'm not sure how to make it simpler. It seems readable to me. Thoughts?